src/main/resources/swagger/api-docs.yaml

```diff
@@ -5268,6 +5285,44 @@ definitions:
         default: false
         description: Commercial purpose/by a commercial entity
 
+  ResearchPurposeRequest:
+    type: object
+    properties:
+      DS:
+        type: array
+        default: []
+        items:
+          type: string
+          example: "http://purl.obolibrary.org/obo/DOID_4"
+        description: >
+          Disease focused research. Values are DOID ontology node ids in the form
+          http://purl.obolibrary.org/obo/DOID_NNN. Specify an empty array if this
+          is not disease focused research.
+      NMDS:
+        type: boolean
+        default: false
+        description: Methods development/Validation study
```

Comment:
in these descriptions, i'd like to include whether the boolean value is indicating something is prohibited or required. For this one: Whether Methods development/Validation study is prohibited.